br/pkg/lightning/backend/kv/base_test.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package kv_test
```

Comment:
```suggestion
package kv
```

It's more easier to let the testing file has same package as the code file, so no need to import `"github.com/pingcap/tidb/br/pkg/lightning/backend/kv"`, and also the testing file can access private functions/variables.

The most common way to have a separate `kv_test` is testing file will import extra pachage and causes a cycle import.

Reply:
got it! good to learn